pwiz_tools/Skyline/CommandArgUsage.resx

```diff
@@ -826,4 +826,106 @@ greater peak area.</value>
   <data name="_chromatograms_limit_noise" xml:space="preserve">
     <value>Limits the length of the chromatagrams to a number of minutes around the chromatogram peak, i.e. ± minutes.</value>
   </data>
+  <data name="_full_scan_acquisition_method" xml:space="preserve">
+    <value>The acquisition method for extracting from full scan data.</value>
+  </data>
+  <data name="_full_scan_isolation_scheme" xml:space="preserve">
+    <value>The isolation scheme used when extracting from full scan DIA data.</value>
+  </data>
+  <data name="_full_scan_precursor_analyzer" xml:space="preserve">
+    <value>The mass analyzer for precursor scans, or "centroided" to use centroid data instead of profile.</value>
+  </data>
+  <data name="_full_scan_precursor_ignore_sim" xml:space="preserve">
+    <value>Skip SIM scans when extracting from full scan MS1 data.</value>
+  </data>
+  <data name="_full_scan_precursor_isotopes" xml:space="preserve">
+    <value>How the precursor threshold is applied when extracting intensities from full scan data.</value>
+  </data>
+  <data name="_full_scan_precursor_isotope_enrichment" xml:space="preserve">
+    <value>The name of the isotope enrichment settings preset to use for extracting from full scan data. If none are available, use the GUI Transition Settings to add a new preset.</value>
+  </data>
+  <data name="_full_scan_precursor_threshold" xml:space="preserve">
+    <value>The number of isotopes or percentage of isotope intensity to extract from full scan data.</value>
+  </data>
+  <data name="_full_scan_product_analyzer" xml:space="preserve">
+    <value>The mass analyzer for product scans, or "centroided" to use centroid data instead of profile.</value>
+  </data>
+  <data name="_import_search_exclude_library_sources" xml:space="preserve">
+    <value>Don't automatically import chromatograms from the result files used to build a library.</value>
+  </data>
+  <data name="_new" xml:space="preserve">
+    <value>Create a new document using the current Skyline settings.</value>
```

Comment:
This seems really error-prone to me to have this using the "current" document settings. I was expecting it would start from the default Skyline settings, so that you always know you are starting from a known baseline. Otherwise, I think users should be starting with a template file where they know that baseline. "Current" can mean whatever the user was last using.